scripts/run_latest_build.sh

```diff
@@ -1,93 +1,49 @@
-#
-# To resolve this, we explicitly set the
-#    --public-hostname and --routing-suffix
-#
-# We use the IP of the docker interface on our host for testing in a
-# local environment, or the external listening IP if we want to expose
-# the cluster to the outside.
-#
-# Below will default to grabbing the IP of docker0, typically this is
-# 172.17.0.1 if not customized
-#
-
-DOCKER_IP="$(ip addr show docker0 2>/dev/null | grep -Po 'inet \K[\d.]+' 2>/dev/null)"
-
-DOCKER_IP=${DOCKER_IP:-"127.0.0.1"}
-PUBLIC_IP=${PUBLIC_IP:-$DOCKER_IP}
-HOSTNAME=${PUBLIC_IP}.nip.io
-ROUTING_SUFFIX="${HOSTNAME}"
-ORIGIN_IMAGE=${ORIGIN_IMAGE:-"docker.io/openshift/origin"}
-ORIGIN_VERSION=${ORIGIN_VERSION:-"latest"}
+TAG=${TAG:-"latest"}
 APB_NAME=${APB_NAME:-"automation-broker-apb"}
 APB_IMAGE=${APB_IMAGE:-"docker.io/automationbroker/automation-broker-apb:latest"}
+BROKER_IMAGE="docker.io/ansibleplaybookbundle/origin-ansible-service-broker:${TAG}"
 BROKER_NAME=${BROKER_NAME:-"ansible-service-broker"}
 BROKER_NAMESPACE=${BROKER_NAMESPACE:-"ansible-service-broker"}
 HELM=${HELM:-"false"}
 
-version=$(oc version | head -1)
-client_version=$(echo $version | egrep -o 'v[0-9]+(\.[0-9]+)+' | tr -d v.)
-if [ "$ORIGIN_VERSION" != "latest" ]; then
-    origin_version=$(echo $ORIGIN_VERSION | tr -d v.)
-    if (( $origin_version > $client_version )); then
-        echo "WARNING: Using client version: $version with cluster version: $ORIGIN_VERSION"
-    fi
-fi
-
-if (( $client_version >= 3100 )); then
-    oc cluster up --image=${ORIGIN_IMAGE} \
-        --tag=${ORIGIN_VERSION} \
-        --enable=service-catalog,template-service-broker,router,registry,web-console \
-        --routing-suffix=${ROUTING_SUFFIX} \
-        --public-hostname=${HOSTNAME}
-else
-    oc cluster up --image=${ORIGIN_IMAGE} \
-        --version=${ORIGIN_VERSION} \
-        --service-catalog=true \
-        --routing-suffix=${ROUTING_SUFFIX} \
-        --public-hostname=${HOSTNAME}
-fi
-
-if [ "$?" -ne 0 ]; then
-    echo "Error starting cluster"
-    exit
-fi
-
-#
-# Logging in as system:admin so we can create a clusterrolebinding and
-# creating ansible-service-broker project
-#
-echo 'Logging in as "system:admin" to create broker resources...'
-oc login -u system:admin
-oc new-project $BROKER_NAMESPACE
+echo "========================================================================"
+echo "                       RUN_LATEST_BUILD"
+echo "========================================================================"
+echo ""
+echo " This script expects a running Kubernetes|OpenShift cluster and a service-catalog."
+echo ""
+echo " OpenShift:"
+echo "   Setup OpenShift: https://github.com/openshift/origin/#installation"
+echo "   You must also enable the service-catalog. This depends on the installed origin"
+echo "   client version."
+echo "     Origin client version < 3.10 \`--service-catalog=true\`."
+echo "     Origin client version >= 3.10 \`--enable=service-catalog\`."
```

Comment:
Question: Do we need PVs installed as well, will it work if you only have just service-catalog?

Reply:
I'm going to try this on an oc cluster up without pvs to check. My initial reaction is no.